app/src/store/data.ts

```diff
@@ -32,9 +32,12 @@ const roundManager = ref<Contract>();
 // Most recent data read is saved as state
 const lastBlockNumber = ref<number>(0);
 const lastBlockTimestamp = ref<number>(0);
+
 const grants = ref<Grant[]>();
+let grantMetadata = ref<Record<string, GrantMetadataResolution>>({});
```

Comment:
Why the change from `const` to `let`? Refs should pretty much always be `const`, not sure if there is a reason to use `let` here